### ISSUE TITLE
Robot Arms

### ISSUE THREAD

**Issue by VermillionX** [GTNewHorizons/NewHorizons#1164]
Robot Arm's function as a cover is to push and pull items from either the inventory of the block/machine it's facing or the one's it's attached to into specific slot: It does NOT do this.
Attached to a Universal Macerator, the robot arm will always take items from the first output slot, regardless of whether the arm is on the macerator or a pipe facing it, and will at random intervals take from the second output slot. Even when configured specifically to take from the macerator's second output slot and output into an adjacent inventory (platinum item pipe) it continues to take from the first output slot.
When attached to the macerator, facing a chest it's possible to output the first output slot into any slot in the chest (useless), but it still doesn't work to output just the second output.
Basically, it's a retarded conveyor, and is shitting on my automation by dumping crushed ores in with secondary output dusts instead of doing its job.

**Comment by Dream-Master** [comment]
Never know the robot arm can used as cover. @Technus any idea about this issue?

**Comment by VermillionX** [comment]
Most GT modules, plates and foils can be used as covers, though only a few actually do anything. You can even mount lenses on the sides of machines.

**Comment by Bryfer** [comment]
On the macerator, as Output?
"**Puts out** into adjancent slot# *" or
"**grabs in** for own slot # *"
And im pretty sure the robotic arm is meant for inputting stuff into machines, rather than output.
![image](https://cloud.githubusercontent.com/assets/22825059/23548103/457f518c-0006-11e7-8145-da7069f27bae.png)

I'd suggest using an Conveyor module instead as output (Or just using the built-in button) and having type/item filters at the input locations for the other machines instead.

**Comment by Namikon** [comment]
> You can even mount lenses on the sides of machines
@Technus Add visible and lethal laser beam if lens attached to laser engraver, thanks

**Comment by VermillionX** [comment]
I've attached the robot arms (MV ones) to both targets: The uni-macerator, the pipe and even adjacent machines with configurations both to "grabs in" and "puts" out in every slot from -1 to +6 in every possible configuration with no success. If it does grab from output slot #2, it doesn't stop there and grabs from slot #1 while it's at it.

**Comment by Bryfer** [comment]
**You are using the Robot Arm cover wrong.**

When you take the "grabs" And..
It's placed on the macerator; You "Grab" from ANY slot of the pipe and **inputs** into slot # you choose of the Macerator.
It's placed on the pipe; You "Grab" from ANY slot of the macerator and **inputs** into slot # you choose of the pipe

When you take the "puts out" And..
It's placed on the macerator; You "Grab" from ANY slot of the macerator and **inputs** into slot # you choose of the pipe.
It's placed on the pipe; You "Grab" from ANY slot of the pipe and **inputs** into slot # you choose of the macerator

**Comment by Technus** [comment]
You can even mount lenses on the sides of machines
@Technus Add visible and lethal laser beam if lens attached to laser engraver, thanks

HOLY YES

**Comment by Technus** [comment]
can i close this?

**Comment by VermillionX** [comment]
Probably.
